api/v1beta1/openstackdataplanedeployment_types.go

```diff
@@ -79,6 +79,12 @@ type OpenStackDataPlaneDeploymentStatus struct {
 
 	// NodeSetHashes
 	NodeSetHashes map[string]string `json:"nodeSetHashes,omitempty" optional:"true"`
+
+	// DeployedVersion
```

Comment:
Is the the version string form OpenStackVersion.Spec.TargetVersion at the point of time when this Deployment was run?

Reply:
It's OpenStackVersion.Spec.TargetVersion at the point when the Deployment is successful, that's when it's read from OpenStackVersion and set here.